Geometry/TrackerGeometryBuilder/test/python/testTrackerModuleInfoDDD_cfg.py

```diff
@@ -25,7 +25,7 @@
 # Choose Tracker Geometry
 if(options.Scenario == _settings.DEFAULT_VERSION):
     print("Loading default scenario: ", _settings.DEFAULT_VERSION)
-    process.load('Configuration.Geometry.GeometryExtended2026Default_cff')
+    process.load('Configuration.Geometry.GeometryExtendedRun4Default_cff')
```

Comment:
need to change here too

https://github.com/cms-sw/cmssw/blob/7d3d80e8707194c87090fd2ef9ae0319ea2cefa1/Geometry/TrackerGeometryBuilder/test/python/testTrackerModuleInfoDD4hep_cfg.py#L28